.pre-commit-config.yaml

```diff
@@ -48,6 +48,7 @@ repos:
     hooks:
     -   id: uv-export
         args: ["--frozen", "--no-hashes", "--no-emit-project"]
+    -   id: uv-sync
```

Comment:
while you are at it, can you update the args above to add `--output-file=requirements.txt`

Reply:
Done 👍